doc/api/child_process.md

```diff
@@ -209,11 +209,10 @@ encoding, `Buffer` objects will be passed to the callback instead.
 const { exec } = require('child_process');
 exec('cat *.js missing_file | wc -l', (error, stdout, stderr) => {
   if (error) {
-    console.error(`exec error: ${error}`);
-    return;
+    console.error(`exit(${error.code}): ${error}`);
   }
-  console.log(`stdout: ${stdout}`);
-  console.error(`stderr: ${stderr}`);
+  console.log('stdout:', stdout);
+  console.error('stderr:', stderr);
```

Comment:
No unrelated changes, please. Can you back this out?

Reply:
@bnoordhuis I was trying to create several examples that show how to do the same thing with different functions, you'll see I used the same code in 326-335: https://github.com/nodejs/node/pull/34345/files#diff-fd8da87c3b5a1cec8e34fc76659ac6a0R326-R335 — is this change still too much, all considered?

Reply:
There's another example that already was using the `console.log('stdout:', stdout);` form — is it more important to leave the other examples unchanged, or can I touch them up for consistency?

Reply:
If it's clearly related to the changes you're making anyway, then it's fine (but even then you should aim to make the most minimal change.) Apparently, it wasn't obvious to me yesterday that they were related. :-)